### DISCUSSION TITLE
Select loaded from array with lazy-loading validation

### DISCUSSION THREAD

**Discussion by CynthiaBlue** [quasarframework/quasar#12335] [CLI - Electron mode]
I have a select that I populate from an array that I want to be a required field.

```
  <q-select
        filled
        :options="peopleArray"
        v-model="eventChair"
        bg-color="indigo-2"
        label="Trial Chairperson" 
        lazy-rules
        :rules="[ val => val && val.length > 0 || 'Field is Required']"
         /> 
```

However, when I use this method, even after the user selects the item from the drop-down, the lazy-rules validation says that the field is not filled in.  Can I use it this way, or do I need to do the validation programmatically instead?  I'm also using it inside a q-form. 

Thanks.

**Comment by MilosPaunovic** [comment]
Everyhing seems in order:
[https://codepen.io/milospaunovic/pen/GROqLjx?editors=1010](https://codepen.io/milospaunovic/pen/GROqLjx?editors=1010)

If you're still experiencing problems, please create a Codepen reproduction.
